themes/bootstrap3/js/covers.js

```diff
@@ -7,9 +7,12 @@ function loadCoverByElement(data, element) {
   function coverCallback(response) {
     spinner.hide();
     container.show();
-    if (response.data.url !== false) {
+    if (response.data.url ?? false !== false) {
```

Comment:
I am a big fan of the nullish coalescing operator (`??`), but unfortunately its support in Javascript doesn't seem to be universal yet. This is causing the Travis build to fail.